src/components/csvLocalizer/jsonLocalizer.js

```diff
@@ -51,20 +76,44 @@ const JsonLocalizer = () => {
             .map(([key, values]) => {
               const cellWidth = 100 / values.length;
               return (
-                <div className="table-row" key={key}>
-                  {/* <div className="cell" style={{ width: `${cellWidth}%` }}>
+                <>
+                  <div className='table-row' key={key}>
+                    {/* <div className="cell" style={{ width: `${cellWidth}%` }}>
                     {key}
                   </div> */}
-                  {values.map((v, i) => (
-                    <div
-                      key={i}
-                      className="cell"
-                      style={{ width: `${cellWidth}%` }}
-                    >
-                      {v}
+                    {values.map((v, i) => (
+                      <Cell
+                        key={`${key}-${i}`}
+                        id={`${key}-${i}`}
+                        style={{ width: `${cellWidth}%` }}
+                        value={v}
+                        onClick={(id) => cellOnClick(id)}
+                        shownInput={shownInput}
+                        className={
+                          shownInput === `${key}-${i}` ? 'active-cell' : ''
```

Comment:
${key}-${i} - is repeated several times. Can we create a variable 'id' let's say?

Reply:
Sure.